src/core_atmosphere/Makefile

```diff
@@ -4,8 +4,11 @@
 # To build a dycore-only MPAS-Atmosphere model, comment-out or delete
 # the definition of PHYSICS, below
 #
-PHYSICS=-DDO_PHYSICS
-
+# If MPAS_CAM_DYCORE is found in CPPFLAGS, PHYSICS will become undefined automatically
+#
+ifeq ($(findstring MPAS_CAM_DYCORE,$(CPPFLAGS)),)
```

Comment:
Although I agree that this is a reasonable change in principle, I think it may be outside the scope what we want to include in the v8.2.1 release. My understanding is that CAM's build already ensured that `DO_PHYSICS` was not defined, and the need for this change may be due to the reuse of makefiles in ongoing work on CAM-SIMA?

Reply:
I fully support creating a separate PR to the `develop` branch with this `MPAS_CAM_DYCORE` logic.